platforms/documentation/docs/src/snippets/plugins/buildscript/kotlin/build.gradle.kts

```diff
@@ -11,12 +11,12 @@ buildscript {
     }
     dependencies {
         classpath("org.yaml:snakeyaml:1.19") // The library's classpath dependency
-        classpath("com.github.johnrengelman:shadow:8.1.1") // The legacy version of Shadow Plugin that needs buildscript
+        classpath("com.gradleup.shadow:shadow-gradle-plugin:8.3.3") // The legacy version of Shadow Plugin that needs buildscript
```

Comment:
I think the comment is misleading here. "Legacy" refers to the legacy way of applying plugins, not to some old version of the Shadow plugin. I'll try to improve the comment instead of reverting this.

Reply:
Indeed